secp256k1-sys/src/lib.rs

```diff
@@ -669,6 +669,271 @@ impl<T> CPtr for [T] {
 #[cfg(fuzzing)]
 mod fuzz_dummy {
     use super::*;
+    use core::sync::atomic::{AtomicUsize, Ordering};
+
+    #[cfg_attr(rust_secp_no_symbol_renaming, error("We do not support fuzzing with rust_secp_no_symbol_renaming"))]
```

Comment:
Is this a thing? don't you need to use `compile_error!(...)` under the cfg?

Reply:
Sure it is - error() has no meaning so it's definitely a compile error :). I'm not sure how to better capture this.

Reply:
sorry for the offtopic but this really reminds me of this comment :) https://github.com/bitcoin-core/secp256k1/pull/787#pullrequestreview-466300677